### ISSUE TITLE
Do a hard shutdown if user presses ctrl-c multiple times

### ISSUE THREAD

**Issue by treeder** [fnproject/fn#710]
Maybe first time, it does soft, but if pressed 5 times, it will hard kill anything running and shutdown. 

Right now, it just waits until it's done serving anything and no way to hard kill it.

**Comment by skinowski** [comment]
is this related #689 where docker receives signal and dies and fn tries to talk to in vain?

**Comment by treeder** [comment]
No, basically if you ctrl-c, then you have to wait for all the hot containers to die off before it quits. I just want a hard quit. The other option would be to just know that there are no requests coming in and just shutdown the hot containers, that would work too.

**Comment by skinowski** [comment]
#771 related if running with dind.
